### ISSUE TITLE
CORS error

### ISSUE THREAD

**Issue by bettybao1209** [nspcc-dev/neofs-http-gw#73]
`Access to fetch at '*' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`

Hi, I try to fetch the object with the http-gateway service in nodejs while it throws the CORS error above. So is it possible to configure the header to make it work?

**Comment by roman-khimov** [comment]
Proper CORS setup is an app-specific thing currently considered to be out of scope for the gateway itself. It can be configured on front-facing proxy server [like in our Medium post](https://neospcc.medium.com/get-your-own-neofs-http-gateway-d076f7169a94).

**Comment by bettybao1209** [comment]
> Proper CORS setup is an app-specific thing currently considered to be out of scope for the gateway itself. It can be configured on front-facing proxy server [like in our Medium post](https://neospcc.medium.com/get-your-own-neofs-http-gateway-d076f7169a94).

Yes, I use the nginx proxy and problem solved.
